rust/tvm-rt/src/to_boxed_fn.rs

```diff
@@ -43,9 +43,8 @@ where
 {
```

Comment:
The whole `ToBoxedFn` seems a little strange. A trait called `To` should have function that takes `Self`, not return `Self`. It seems to be implemented backwards.

Could you instead just have a generic, inherent method on `Function`?

Reply:
Ah nvm, this is unrelated to this change.